### PR TITLE
catgirls and lizards get no disease from rats

### PR DESCRIPTION
**Pull request by MCterra10**
## About The Pull Request
Title

## Why It's Good For The Game
It makes sense lmao

## Changelog
:cl:
tweak: catgirls and lizards no longer get disease from rats
/:cl:

### CODE REVIEW

File: austation/code/modules/mob/living/carbon/human/species_type/felinid.dm [hunk 1]
```diff
@@ -1,2 +1,6 @@
 /datum/species/human/felinid
 	inert_mutation = /datum/mutation/human/claws // Claws
+
+/datum/species/human/felinid/New()
+	inherent_traits.Add(TRAIT_STRONG_STOMACH)
+	. = ..()
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/carbon/human/species_type/felinid.dm:6]:
```suggestion
	return ..()
```

File: austation/code/modules/mob/living/carbon/human/species_type/lizardpeople.dm [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/datum/species/lizard/New()
+	inherent_traits.Add(TRAIT_STRONG_STOMACH)
+	. = ..()
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/carbon/human/species_type/lizardpeople.dm:3]:
```suggestion
	return ..()
```

File: austation/code/__DEFINES/diseases.dm [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Custom spread flag that prevents disease being caught by catgirls/lizards when *ingested*
+#define DISEASE_SPREAD_STRONG_STOMACH	(1<<7)
```
Comment (TheFakeElon) [inline austation/code/__DEFINES/diseases.dm:2]: Name is a bit misleading, I had to read the flag toggle a few times to understand, but maybe that's just me. 

Maybe specifying immunity in the name might help a bit? i.e `DISEASE_CHECK_STRONG_STOMACH`

**Review by TheFakeElon — APPROVED** [review]

**Comment by amidoingitright** [comment]
uh oh
stinky

**Pull request merged**